src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/DatabaseErrorPageMiddleware.cs

```diff
@@ -69,6 +69,16 @@ public DatabaseErrorPageMiddleware([NotNull] RequestDelegate next, [NotNull] ISe
 
             try
             {
+#if NET461
+                // TODO This probably isn't the correct place for this workaround, it
```

Comment:
No idea. I vaguely remember something about the response's content type not being set correctly without this.

It was written ages ago and has probably since been fixed. I say we remove it and see what breaks.